cnxpublishing/tests/views/test_admin.py

```diff
@@ -232,3 +239,114 @@ def test_site_messages_edit_post(self):
+        cls.db_conn_str = cls.settings[CONNECTION_STRING]
+        cls.db_connect = staticmethod(db_connection_factory())
+
+    def setUp(self):
+        self.config = testing.setUp(settings=self.settings)
+        self.config.include('cnxpublishing.tasks')
+        init_db(self.db_conn_str, True)
+        add_data(self)
+
+        # Set up routes
+        from ... import declare_api_routes
+        declare_api_routes(config)
+
+    def tearDown(self):
+        with self.db_connect() as db_conn:
+            with db_conn.cursor() as cursor:
+                cursor.execute("DROP SCHEMA public CASCADE")
+                cursor.execute("CREATE SCHEMA public")
+        testing.tearDown()
+
+    def test_admin_content_status_no_filters(self):
+        request = testing.DummyRequest()
+
+        from ...views.admin import admin_content_status
+        content = admin_content_status(request)
+        self.assertEqual({
+            'SUCCESS': 'checked',
+            'PENDING': 'checked',
+            'STARTED': 'checked',
+            'RETRY': 'checked',
+            'FAILURE': 'checked',
+            'start_entry': 0,
+            'page': 1,
+            'num_entries': 100,
+            'sort': 'bpsa.created DESC',
+            'sort_created': 'fa fa-angle-down',
+            'states': content['states']
+        }, content)
+        self.assertEqual(len(content['states']), 2)
+        self.assertEqual(
+            content['states'],
+            sorted(content['states'], key=lambda x: x['created'], reverse=True))
+
+    def test_admin_content_status_w_filters(self):
+        request = testing.DummyRequest()
+
+        request.GET = {'page': 1,
+                       'number': 2,
+                       'sort': 'STATE ASC',
+                       'author': 'charrose',
+                       'status_filter': 'PENDING'}
+        from ...views.admin import admin_content_status
+        content = admin_content_status(request)
+        self.assertEqual({
+            'PENDING': 'checked',
+            'start_entry': 0,
+            'page': 1,
+            'num_entries': 2,
+            'author': 'charrose',
+            'sort': 'STATE ASC',
+            'sort_state': 'fa fa-angle-up',
+            'states': content['states']
+        }, content)
+        self.assertEqual(len(content['states']), 2)
+        for state in content['states']:
+            self.assertTrue('charrose' in state['authors'])
+            self.assertTrue(state['state'] == 'PENDING')
+        self.assertEqual(
+            content['states'],
+            sorted(content['states'], key=lambda x: x['state']))
+
+    def test_admin_content_status_single_page(self):
+        request = testing.DummyRequest()
+
+        uuid = 'd5dbbd8e-d137-4f89-9d0a-3ac8db53d8ee'
+        request.matchdict['uuid'] = uuid
+
+        from ...views.admin import admin_content_status_single
+        content = admin_content_status_single(request)
+        print(content)
```

Comment:
Sorry, what I was really asking for is to have these removed. No need to add to the wall of text with additional print lines. 😜